plugin_xiplot_filetypes/README.md

```diff
@@ -0,0 +1,5 @@
+# [&chi;iplot](https://github.com/edahelsinki/xiplot) plugin for additional file types
+
+This plugin adds support for additional file types (beside `csv` and `json`) to [&chi;iplot](https://github.com/edahelsinki/xiplot).
+Currently, this plugin adds support for `feather` and `parquet`.
+Note that in the [WASM version](https://edahelsinki.fi/xiplot) only `parquet` is added.
```

Comment:
There has been some recent activity in https://github.com/pyodide/pyodide/issues/2933, so getting pyarrow into Pyodide doesn’t seem too far away anymore. So while this restriction is ok for now, we should keep an eye on that issue and remove the limitation once possible.

Reply:
nice